Core/clim-basic/mirrors.lisp

```diff
@@ -176,10 +176,16 @@ very hard)."
   (let* ((parent (sheet-parent sheet))
          (sheet-region-in-native-parent
            ;; this now is the wanted sheet mirror region
-           (transform-region (sheet-native-transformation parent)
+          (if (graftp parent)
+              ;; for the TOP-LEVEL-SHEET-PANE (when the parent is a GRAFT) we don't
+              ;; clip the sheet-region with the parent-region.
```

Comment:
please add your initials and date after the comment. Also start the comment with capital letter. I.e

;; Foo goes beyond QUX so we do BAR. -- admich 2024-11-24